df.world.xml

```diff
@@ -948,13 +948,13 @@
             <stl-string name='last_name_seed'/>
             <stl-string name='last_history_seed'/>
             <stl-string name='last_creature_seed'/>
-            <bool name='placed_caves'/>
-            <bool name='placed_good_evil'/>
-            <bool name='placed_megabeasts'/>
-            <bool name='placed_other_beasts'/>
-            <bool name='made_cave_pops'/>
-            <bool name='made_cave_civs'/>
-            <bool name='placed_civs'/>
+            <bool name='place_caves'/>
+            <bool name='place_good_evil'/>
+            <bool name='place_megabeasts'/>
+            <bool name='place_other_beasts'/>
+            <bool name='make_cave_pops'/>
+            <bool name='make_cave_civs'/>
+            <bool name='place_civs'/>
```

Comment:
The former names seem appropriate here - that is, some of these flags are false until the corresponding steps in worldgen run.

Reply:
Looks like nothing actually uses these fields, so fine with me. Your observations appear to be correct for `place_caves` and `place_megabeasts`, at least.